Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -114,6 +114,16 @@ Restricted users created using `--restricted-auth-permission` flag need to pass
     An existing non-restricted user cannot be converted to a restricted user by upgrading.
     The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
 
+#### Upgrade and utilize new feature
+
+User need to re-run the python scripts with the same flags that were used during installation plus the new flags they want to use
+And after that [copy the bash output](#2-copy-the-bash-output) run the [import script](#import-the-source-data).
+
+User can find there old flags configuration in the `external-cluster-user-command` configmap `command` data.
+
+!!! note
+    User should only change flag value if they want to force change there configuration.
```

Comment:
IIRC, we've been trying to avoid use of the word "user" after the Ceph doc audit of our docs. 

I also think that the consideration of when to run the script again is important enough that it can be a warning and not merely a 'note'.

Also, let's be sure to explain why/when users care about reading this doc section.

And we should be aware that users don't know the configmap was only added in rook v1.15. If v1.15 users try to look for the configmap and don't find it, we don't need to get bug reports about it.

This can be reworded like:
> Some Rook upgrades may require re-running the import steps, or may introduce new external cluster features that can be most easily enabled by re-running the import steps.
> 
> To re-run the import steps with new options, the python script should be re-run using the same configuration options that were used for past invocations, plus the configurations that are being added or modified. 
>
> Starting with Rook v1.15, the script stores the configuration in the `external-cluster-user-command` configmap for easy future reference.
>
> !!! warning
>     Failure to reuse the same configuration options when re-invoking the pythong script can result in unexpected changes when re-running the import script.

Reply:
Thanks, it looks pretty nice